Gemfile

```diff
@@ -14,5 +14,11 @@ gemspec
 # gem 'byebug', group: [:development, :test]
 group :test  do
```

Comment:
I think this needs to be Test and development because we have removed the dev dependency in the gemspec file and so none of these would be added if bundle install was run on dev mode